Java/avg/array.java

```diff
@@ -12,4 +12,15 @@ public static void main(String[] args) {
             System.out.println(arr[i]);
         }
     }
+
+    public static void Even(String[] args) {

```

Comment:
**Suggestion**: Rename the Even method to follow Java naming conventions, using lowerCamelCase, such as 'printEvenNumbers'. <br> **Code Suggestion**: 
 ```
 -    public static void Even(String[] args) {
 +    public static void printEvenNumbers(String[] args) {
 ```